docs/.vitepress/config.ts

```diff
@@ -338,7 +338,7 @@ export default defineConfig({
       ],
       '/changes/': [
         {
-          text: 'Major Changes',
+          text: 'Breaking Changes',
```

Comment:
One reason to call it "Major Changes" was to avoid the need to force us to document every Breaking Change in the same way. It is kind of "Major Breaking Changes". But probably having breaking here is more clear and we don't need to feel the pressure to have small breaking changes guides.

Reply:
That's true, maybe we can mention not all breaking changes are listed here? Or do you think we should still stick with major changes?

Reply:
Let's merge and then we can continue refining this later. I think we should at least mention that only major breaking changes will be documented in this way